File: app/src/main/java/com/druger/aboutwork/interfaces/view/NetworkView.java
package com.druger.aboutwork.interfaces.view;

import com.druger.aboutwork.enums.TypeMessage;

/**
 * Created by druger on 06.08.2017.
 */

public interface NetworkView {

    void showProgress(boolean show);

    void showMessage(String message, TypeMessage typeMessage);
}

File: app/src/main/java/com/druger/aboutwork/interfaces/view/NetworkView.kt
package com.druger.aboutwork.interfaces.view

import androidx.annotation.StringRes
import com.druger.aboutwork.enums.TypeMessage

/**
 * Created by druger on 06.08.2017.
 */

interface NetworkView {

    fun showProgress(show: Boolean)

    fun showMessage(@StringRes message: Int, typeMessage: TypeMessage)

    fun showMessage(message: String)

    fun showErrorScreen(show: Boolean)
}
